src/components/InputField/InputField.styled.js

```diff
@@ -0,0 +1,64 @@
+import styled from 'styled-components';
+
+const getNotificationColor = ({ $isErrorColor, $isSuccessColor }) =>
+{
+  if ($isErrorColor) return 'var(--color-error)';
+  if ($isSuccessColor) return 'var(--color-success)';
+  return '#79747E';
+};
+
+const generalNotificationStyles = `
+  font-size: 12px;
+  text-align: left;
+`;
+
+export const InputFieldStyled = styled.div`
+  margin-bottom: 15px;
+  position: relative;
+`;
+
+export const StyledLabel = styled.label`
+  color: ${getNotificationColor};
+`;
+
+export const StyledInput = styled.input`
+  width: calc(100% - 16px);
+  font-size: 14px;
+  height: 56px;
+  border-radius: 8px;
+  border: 1px solid;
+  border-color: ${getNotificationColor};
+
+  &:hover {
+    border: 1px solid #111;
+  }
+
+  &::placeholder {
+    color: #49454f;
+    position: absolute;
```

Comment:
You can't position element(pseudo element) which is not in DOM. https://css-tricks.com/almanac/selectors/p/placeholder/#article-header-id-4